src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -1105,6 +1107,26 @@ public String run(final Matcher m) {
 					hasDelayBefore = hadDelayBeforeLastIf;
 					((Conditional) items.get(items.size() - 1)).loadElseIf(cond, (SectionNode) n);
 					hasDelayBefore = hadDelayBeforeLastIf.or(hadDelayAfterLastIf.and(hasDelayBefore.and(Kleenean.UNKNOWN)));
+				} else if (StringUtils.startsWithIgnoreCase(name, "parse if ")) {
+					name = name.substring("parse if ".length());
+					final Condition cond = Condition.parse(name, "can't understand this condition: '" + name + "'");

```

Comment:
the error should have a capital C

Reply:
While I 100% agree, I was keeping consistency with the rest of it:
<img width="1651" alt="Screen Shot 2020-09-24 at 12 50 12 PM" src="https://user-images.githubusercontent.com/20008482/94192976-8f256a00-fe64-11ea-9bdb-62c6a943feeb.png">